pkg/k8shandler/logstore.go

```diff
@@ -108,9 +108,6 @@ func getElasticsearchCR(logging *logging.ClusterLogging, elasticsearchName strin
 		Roles:        []v1alpha1.ElasticsearchNodeRole{"client", "data", "master"},
```

Comment:
For the purpose of Cluster logging, we only allow CDM nodes?
If yes, then this PR is good.